spring-boot-project/spring-boot/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -68,11 +68,21 @@
       "sourceType": "org.springframework.boot.context.logging.LoggingApplicationListener"
     },
     {
-      "name": "logging.file",
+      "name": "logging.file.name",
       "type": "java.lang.String",
       "description": "Log file name (for instance, `myapp.log`). Names can be an exact location or relative to the current directory.",
       "sourceType": "org.springframework.boot.context.logging.LoggingApplicationListener"
     },
+    {
+      "name": "logging.file",
+      "type": "java.lang.String",
+      "description": "Log file name (for instance, `myapp.log`). Names can be an exact location or relative to the current directory.",
+      "sourceType": "org.springframework.boot.context.logging.LoggingApplicationListener",
+      "deprecation": {
+        "replacement": "logging.file.name",
+        "level": "error"
```

Comment:
Sweet 👌

Reply:
I removed `"level": "error"`. This should do as you suggested, right? Or is there more steps that need to be taken for the value of the old `logging.file` property to be copied over to the new `logging.file.name` property?

Reply:
That's only flagging the kind of deprecation of the property. If you switch from error to warning, the code has to read the old property has a fallback. Check the place in your PR where you've changed the property name. 

That said, we haven't make up our mind about that so please hold on until we figure this out.

Reply:
Alright, no problem. When the decision has been made, let me know and I'll make the change accordingly.